docs/reference/commandline/daemon.md

```diff
@@ -33,6 +33,7 @@ weight = -1
       --dns-opt=[]                           DNS options to use
       --dns-search=[]                        DNS search domains to use
       --default-ulimit=[]                    Set default ulimit settings for containers
+      --escape                               Sets the default escape character for a Dockerfile
```

Comment:
This is not "a Dockerfile", right? It's the default for all Dockerfiles that are going to get built by this engine.